en/admin/scripts.rst

```diff
@@ -0,0 +1,113 @@
+
+:meta-keywords: scripts
+:meta-description: Various CUBRID scripts support CUBRID Managements
+
+***************
+cubrid scripts
+***************
+
+.. _cubrid_scripts_unloaddb_sh:
+
+unloaddb.sh script
+==================
+
+To shorten the execution time of **unloaddb** of a large size database, it is possible to run **cubrid unloaddb** processes concurrently over groups of different tables.
+**unloaddb.sh** is a shell script that automates this process.
+
+To evenly distribute unloaddb load over multiple child processes, the **unloadb.sh** gets the size information of the tables to be unloaded from the system catalog (this requires dba privileges).
+
+Tables to be unloaded are allocated to each child process based on the table size, and each child process simultaneously unloads the allocated tables.
+
+
+.. warning::
+
+	**restrictions:**
+	  * Require Linux Environments
+	  * Only users with **DBA privileges** can run this script
+
+The following is [options] used in **unloaddb.sh**.
+
+.. program:: sh unloaddb.sh
+
+.. option:: -u user
+
+    Specify a user account of a database to be unloaded. If this is not specified, the default is **DBA**. ::
+
+        sh unloaddb.sh -u dba demodb
+
+.. option:: -t no-process
+
+    Number of child processes to run concurrently (default: 8, maximum 16) ::
+
+        sh unloaddb.sh -4 demodb
+
+.. option:: -i input-class-file
+
+    Unload schemas, indexes and data of specified tables in the file. (default: all tables in the database) ::
+
+        sh unloaddb.sh -i table_list.txt demodb
+
+    The following example shows an input file (table_list.txt). ::
+
+        table_1
```

Comment:
From v11.2, cubrid supported user schema feature.
So, you need to replace table_1 to user.table_1 at the manual.
If it haven't user name in the table name, it is need to display error at v11.2 or later.

Reply:
Thank you for your comment, I missed '**user schema**' !

Reply:
It was fixed.